src/plan_partialize.h

```diff
@@ -3,7 +3,11 @@
  * Please see the included NOTICE for copyright information and
  * LICENSE-APACHE for a copy of the license.
  */
+#ifndef TIMESCALEDB_PLANNER_PARTIALIZE_H
```

Comment:
hmm i think this should be TIMESCALEDB_PLAN_PARTIALIZE_H to be consistent

Reply:
Oops, yep